integrations/rails.md

```diff
@@ -35,7 +35,7 @@ This procedure creates a [basic Ruby on Rails application](https://guides.rubyon
         ],
           toolbar: 'undo redo | formatselect | ' +
```

Comment:
Another set of duplicate spaces here?

Reply:
fixed

Reply:
https://github.com/tinymce/tinymce-docs/pull/1426/commits/49f17aabc2e94afcff276b9e332dab05e39d67ae